client/packages/common/src/ui/layout/tables/material-react-table/components/ChipTableCell.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Box, Chip } from '@openmsupply-client/common';
+import { MRT_Cell, MRT_RowData } from 'material-react-table';
+
+export const ChipTableCell = <T extends MRT_RowData>({
```

Comment:
There was already a componet for this under patients, I just copied it over an tweaked slightly.
I wonder if we should have a max width for the chip with tooltip to show the full name for very long master list names?